tests/feeder/tests.cpp

```diff
@@ -136,4 +136,4 @@ TEST_F(ETFeederTest, NodeGetChildrenTest) {
 int main(int argc, char** argv) {
   ::testing::InitGoogleTest(&argc, argv);
   return RUN_ALL_TESTS();
-}
+}
```

Comment:
This change is minor, but it's unnecessary. Please avoid introducing unnecessary changes that are unrelated to this PR.